.circleci/config.yml

```diff
@@ -164,7 +171,7 @@ workflows:
               only: /.*/
             branches:
               # Note to reviewers: will change to develop before merging, once build works off the branch
-              only: feature/SEAB-314/cd
+              only: /.*/
```

Comment:
This and line 181 were intended to be `develop`. Line 181 should definitely be develop (I overlooked it when I merged). This one could arguably continue to be its current value; it might be useful to have have builds from all branches on s3.

Reply:
This is the distro for cloud.dockstore.org right?
TBH,  I don't know that "develop(no hash)" is useful anymore. 

At least with the current approach, in compose the VM always has a specific hash fragment for a GUI deployment that it is pointing at on gui.dockstore.org. If we were to build a CI/CD, I seems like we should just point to a deployment with a correct branch/tag-hash value.

Reply:
> This is the distro for cloud.dockstore.org right?

Yes, this is for cloud.dockstore.org, whose whole approach, I just realized, probably doesn't make sense any more given the way we are generating the UI -- why have have an index.html on cloud.dockstore.org in turn load files from gui.dockstore.org?

I do think we should come back to it, so for now I think we should not execute the CF invalidation any more (which is what you say in your other comment).